HGGM.UnitTests/ProgramTests.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using FluentAssertions;
+using Xunit;
+
+namespace HGGM.UnitTests
+{
+    public class ProgramTests
+    {
+        [Fact]
+        public void ProjectExists()
+        {
+            new Program().Should().NotBeNull();
+        }
+
+        [Fact]
+        public void WebHostBuilds()
+        {
+            Program.CreateWebHostBuilder(new string[0]).Build().Should().NotBeNull();
+        }
+    }
```

Comment:
Two simple tests that don't really test anything, placeholders to test build reporting